static/docs/user-guide/external-dependencies.md

```diff
@@ -101,30 +101,28 @@ credentials for certain remotes.
 
 ## Example: Using import-url
 
-In the previous command examples, downloading commands were used: `aws s3 cp`,
-`scp`, `wget`, etc. `dvc import-url` simplifies the downloading part for all the
-supported types of dependencies.
+In the previous examples, downloading commands were used: `aws s3 cp`, `scp`,
+`wget`, etc. `dvc import-url` simplifies the downloading for all the supported
+types of dependencies.
 
 ```dvc
 $ dvc import-url https://dvc.org/s3/get-started/data.xml
+Importing 'https://dvc.org/s3/get-started/data.xml' -> 'data.xml'
+[##############################] 100% data.xml
+...
 ```
 
-<details>
+The command above creates an <abbr>import stage</abbr> specified in DVC-file
+`data.xml.dvc` that uses an external dependency (in this case of HTTP type).
 
-### Expand to learn more about DVC internals
+<details>
 
-The resulting DVC-file will contain something like this:
+### Expand to see the (partial) resulting DVC-file
```

Comment:
why partial?

Reply:
Because I'm only showing the `deps` section of the DVC-file. I guess the wording is weird. Changed to "Expand to see resulting dependencies in the DVC-file" in 6ec8f30.

What do you think?

Reply:
yeah, I understand. I mean why do we want to show partials only. I think we can certainly remove some default values but I would show more or less all parts of the file.

Reply:
or we can put `...` in the YAML block or even `(not relevant parts are omitted)` ... but again, I would use it carefully and try to keep the high level structure for the file - it can give more intuition to a user

Reply:
True. I guess we don't use many partial DVC-file code blocks in other docs. I restored more complete versions with `...` instead of initial default values in 93f6fe7